SYCL/ESIMD/dpas/dpas_int.cpp

```diff
@@ -0,0 +1,58 @@
+//==---------------- dpas_int.cpp  - DPC++ ESIMD on-device test ------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu-intel-pvc || esimd_emulator
+// UNSUPPORTED: cuda || hip
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+
+// This test verifies DPAS support for 2,4,8-bit integers.
+
+#include "dpas_common.hpp"
+
+int main(int argc, const char *argv[]) {
+  queue Q(esimd_test::ESIMDSelector{}, esimd_test::createExceptionHandler());
+
+  bool Print = argc > 1 && std::string(argv[1]) == "-debug";
+  bool Passed = true;
+
+  // Test unsigned 2-bit integers./
+  Passed &= tests<8, 8, u2, u2>(Q, Print);
+  Passed &= tests<8, 4, u2, u2>(Q, Print);
+  // TODO: enable this case when the problem with simd constructor
```

Comment:
Is there internal tracker on this problem?

Reply:
Yes, I created one.